src/queue.jl

```diff
@@ -4,6 +4,15 @@ type Queue{T}
     store::Deque{T}
 end
 
+"""
+    Queue{T}(ty::Type{T}, blksize::Integer)
```

Comment:
Here again I think you can write `ty::Type{T}` as `T`, and get rid of the type parameter.